package.json

```diff
@@ -21,7 +21,7 @@
   "main": "src/index.js",
   "repository": "github:multiformats/js-multibase",
   "scripts": {
-    "prepare": "aegir build",
+    "prepare": "aegir build --no-bundle",
```

Comment:
Do we want to keep the `prepare` script around? Or only when in development for testing purposes?

Reply:
It believe it is in place to use git url in dependent packages.

Reply:
doesnt hurt i guess, dunno maybe we should discuss it